flang/runtime/numeric-templates.h

```diff
@@ -242,10 +242,13 @@ inline RT_API_ATTRS T RealMod(
         IS_MODULO ? "MODULO with P==0" : "MOD with P==0");
   }
   if (ISNANTy<T>::compute(a) || ISNANTy<T>::compute(p) ||
-      ISINFTy<T>::compute(a)) {
+      ISINFTy<T>::compute(a) ||
+      // Other compilers behave consistently for MOD(x, +/-INF)
```

Comment:
This would be more clear as a distinct `else if` block.

Reply:
Thanks! Done.